src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -1043,8 +1043,10 @@ aida:RdfStatementShape
             aida:EventArgumentSubclass
             aida:RelationArgumentSubclass ) ;
         sh:message "All rdf:Statements must be type declarations or have predicates from the domain relation or event argument types. This error is often caused by using a value outside the domain ontology"
-    ]
-    .
+    ] ;
+
+    # may provide a importance
+    sh:property aida:ImportancePropertyShape .
```

Comment:
This isn't bad, but I don't think we're allowing aida:importance on type assertions. You should probably put this on EventArgumentShape and RelationArgumentShape.

Reply:
fixed.